precise/scripts/train.py

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 from fitipy import Fitipy
-from keras.callbacks import LambdaCallback
+from tensorflow.keras.callbacks import LambdaCallback
```

Comment:
This might have been a mistake.
Should keep Keras callbacks directly from module (instead of tf.keras).

Reply:
As I mentioned, I think [it says here](https://www.pyimagesearch.com/2019/10/21/keras-vs-tf-keras-whats-the-difference-in-tensorflow-2-0/) it sounds like `tensorflow.keras` is recommended in Tensorflow 2.

Reply:
Actually, both libraries are being used, just not at the same time.
For training, Keras is used. For conversion tf.keras is used.

I agree tf.keras should be prefered, however there's a difference in the implementations for Sequence and GRU between Keras and tf.keras. This is why the Keras module is used for creating/training the model. Once the model has been saved to a .net file, tf.keras is used to allow compat with tf.lite.converter. (This way, we avoid backend collisions between keras and tf.keras)

Reply:
Addressed this is latest commit 'General cleanup for merge'.